dev/create-release/spark-rm/Dockerfile

```diff
@@ -43,7 +43,7 @@ ARG PIP_PKGS="pyopenssl pypandoc numpy pygments sphinx"
 # This is all in a single "RUN" command so that if anything changes, "apt update" is run to fetch
 # the most current package versions (instead of potentially using old versions cached by docker).
 RUN apt-get clean && apt-get update && $APT_INSTALL gnupg && \
-  echo 'deb http://cran.cnr.Berkeley.edu/bin/linux/ubuntu bionic-cran35/' >> /etc/apt/sources.list && \
+  echo 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/' >> /etc/apt/sources.list && \
```

Comment:
I couldn't test that this works as a Debian repo, but the result of browsing both URLs looks the same and contains a bionic-cran35 subdir.

Reply:
This seems to need more works. Could you try with docker image?
```
root@8a08d6e45096:/# apt-get update
Ign:1 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ InRelease
Err:2 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ Release
  Certificate verification failed: The certificate is NOT trusted. The certificate issuer is unknown.  Could not handshake: Error in the certificate verification. [IP: 54.192.146.13 443]
```

Reply:
Dang, OK, I didn't test this. I don't see browser warnings when browsing to https://cloud.r-project.org/bin/linux/ubuntu so that's odd. How are you testing it? What about using https://cran.r-project.org/bin/linux/ubuntu directly here?

Reply:
I tested like this.
```bash
$ docker run -it ubuntu:18.04 /bin/bash
root@f088bff95325:/# apt-get clean && apt-get update && apt-get install --no-install-recommends -y gnupg && echo 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/' >> /etc/apt/sources.list && gpg --keyserver keyserver.ubuntu.com --recv-key E084DAB9 && gpg -a --export E084DAB9 | apt-key add - && apt-get clean && rm -rf /var/lib/apt/lists/* && apt-get clean && apt-get update
```

Reply:
See my latest commit. Adding `ca-certificates` and `[trusted=yes]` made it work for me.

Reply:
Great!